packages/library/src/commonMain/kotlin/io/realm/internal/RealmLog.kt

```diff
@@ -0,0 +1,87 @@
+package io.realm.internal
+
+import io.realm.LogConfiguration
+import io.realm.log.LogLevel
+import io.realm.log.RealmLogger
+
+/**
+ * Logger class used by Realm components. One logger is created for each Realm instance.
+ */
+// FIXME Only public because of testing. Using @Suppress("invisible_members") is not enough.
```

Comment:
I think it is `invisible_member`. There is also a similar `invisible_reference`, which is probably what you would need here. ... or both